src/main/resources/assets/computercraft/lua/rom/programs/edit.lua

```diff
@@ -31,12 +31,12 @@ local bRunning = true
 -- Colours
 local highlightColour, keywordColour, commentColour, textColour, bgColour, stringColour
 if term.isColour() then
```

Comment:
On both normal and advanced computers, it needs to load colours from setings, *but* the defaults need to be different for each! On a normal computer, I want my directory name to be white, not whatever shade of gray green comes out to.